### ISSUE TITLE
sharedIndexInformer warning and UI oddity

### ISSUE THREAD

**Issue by windowsrefund** [fjogeleit/policy-reporter#160]
Seeing 2 things after upgrading to 2.6.1 along with Kyverno 1.7.0.

```
2022-06-21T15:57:50.076802254-04:00 W0621 19:57:50.076713       1 shared_informer.go:401] The sharedIndexInformer has started, run more than once is not allowed
```

In the UI, I only see data related to a single namespace.

Happy to provide additional information as needed.

**Comment by windowsrefund** [comment]
2.6.2-rc1 fixes this issue.

**Comment by fjogeleit** [comment]
I released v2.9.4 of the Helm Chart with the fixed version 2.6.2 of policy-reporter

**Comment by fjogeleit** [comment]
Because of an issue in the Helm Chart, v2.9.4 still used the old version. I released v2.9.5 with the working one.
